include/ck/tensor_operation/gpu/device/impl/device_put_element_impl.hpp

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
```

Comment:
Add some comments to indicate this file is not used

Reply:
I add the example